### ISSUE TITLE
Water waves are too high

### ISSUE THREAD

**Issue by pallebone** [depressed-pho/natural-mystic-shaders#78]
Hi, 

Thank you for this nice texture pack. one issue is the ware when it waves appears to wave too high making the ware higher than adjacent blocks. This effect looks strange as you see the edge of the ware higher than blocks next to water. (overlapping issues).

Let me know if you need a screenshot.

Pete

**Comment by depressed-pho** [comment]
Yes I'm aware of the problem. But for now I can't fix it, because the game engine doesn't tell shaders anything about the shape of terrain.
